docker-compose.yml

```diff
@@ -7,15 +7,19 @@ services:
     volumes:
       - .:/go/src/app/bino
     working_dir: /go/src/app/bino
-    #command: go run main.go
-    ports: 
+    environment:
+      - BINO_DISCORD_TOKEN
+    command: go run main.go
+    ports:
```

Comment:
we don't need to expose the port as we won't be using it outside the container.

Reply:
fixed